src/quic/frame_writer.d

```diff
@@ -114,5 +114,5 @@ unittest
     LocalAppender buffer;
     writer.getBytes(buffer, packet);
     assert(buffer[].toHexString!(LetterCase.lower) ==
-                                "c00000000108000102030405060705635f63696400");
+                                "c00000000108000102030405060705635f6369640000");
```

Comment:
Thanks! You are right, your test is better. The additional 0 byte would mean that packetPayload is empty.